mediator/fd_cesm.yaml

```diff
@@ -318,6 +318,10 @@
        canonical_units: 1e-6 mol/mol
        description: atmosphere export - prognostic CO2 at the lowest model level
      #
+     - standard_name: Sa_o3
+       canonical_units: 1e-9 mol/mol
+       description: atmosphere export - prognostic CO2 at the lowest model level
```

Comment:
Should this be something like "atmosphere export - O3 in the lowest model layer (prognosed or prescribed)"?

Reply:
Thanks for the catch. I have fixed this.